plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -493,6 +676,15 @@ protected boolean isL3Network(Long offeringId) {
                 || _ntwkOfferingSrvcDao.areServicesSupportedByNetworkOffering(offeringId, Service.StaticNat);
     }
 
+    private String getPreConfiguredDomainTemplateName(Network network, NetworkOffering networkOffering) {
+        if (network.getVpcId() != null) {
+            return _configDao.getValue(NuageVspManager.NuageVspVpcDomainTemplateName.key());
+        } else if (networkOffering.getGuestType() == Network.GuestType.Shared) {
+            return _configDao.getValue(NuageVspManager.NuageVspSharedNetworkDomainTemplateName.key());
+        }
+        return _configDao.getValue(NuageVspManager.NuageVspIsolatedNetworkDomainTemplateName.key());
```

Comment:
Why not set a local variable with the value to passed into ``_configDao.getValue(..)`` to make the method more concise and DRY?